src/lib.rs

```diff
@@ -384,7 +384,7 @@ impl Maildir {
     /// Stores the given message data as a new message file in the Maildir `new` folder. Does not
     /// create the neccessary directories, so if in doubt call `create_dirs` before using
     /// `store_new`.
```

Comment:
Please update the documentation on this and on `store_new` to indicate that it returns the generated id of the new message.

Reply:
Done